TESTS/mbed_drivers/lp_timeout/main.cpp

```diff
@@ -74,10 +74,13 @@ Case cases[] = {
     Case("1 s delay during deepsleep (attach_us)", test_deepsleep<AttachUSTester<LowPowerTimeout>, 1000000, LONG_DELTA_US>,
          greentea_failure_handler),
 #endif
+    /* Time drifting tests are unstable on CI. Disable them for now. */
+#if TIME_DRIFT_TESTS
```

Comment:
So this means all time drifting tests will be turned off by default? We can only enable them by put `-DTIME_DRIFT_TESTS` when compile the greentea tests ?